src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  font-family: Roboto, system-ui;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
The font-weight might be better placed in a more specific selector or in a class that specifically handles text styling. It's not common to see text styling in a container class like this.